akka-stream/src/main/scala/akka/stream/javadsl/StreamRefs.scala

```diff
@@ -28,7 +26,6 @@ object StreamRefs {
    *
    * See more detailed documentation on [[SourceRef]].
    */
-  @ApiMayChange
   def sourceRef[T](): javadsl.Sink[T, CompletionStage[SourceRef[T]]] =
```

Comment:
Let's try to do https://github.com/akka/akka/issues/24372 before removing the ApiMayChange for StreamRefs. That should be a separate PR, so just revert in this file here.

Reply:
I have fixed StreamRefs in https://github.com/akka/akka/pull/26847

Reply:
OK, I will update docs.